src/client/Microsoft.Identity.Client/Credential/CredentialResponse.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+#if SUPPORTS_SYSTEM_TEXT_JSON
+using Microsoft.Identity.Client.Platforms.net6;
+using JsonProperty = System.Text.Json.Serialization.JsonPropertyNameAttribute;
+#else
+using Microsoft.Identity.Json;
+#endif
+
+namespace Microsoft.Identity.Client.Credential
+{
+    [JsonObject]
+    [Preserve(AllMembers = true)]
+    internal class CredentialResponse
```

Comment:
Could these be part of the "Feature" folder? It's a bit strange to put a single file in the "Features" folder.